tests/manage/z_cluster/test_flexible_scaling_ui.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_no_lso,
+    tier1,
+    vsphere_platform_required,
+    skipif_ui_not_support,
+    bugzilla,
+    ui,
+    ignore_leftovers,
+)
+from ocs_ci.ocs.resources.pod import get_osd_pods
+from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.add_replace_device_ui import AddReplaceDeviceUI
+from ocs_ci.ocs.node import get_nodes, add_disk_to_node
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pv import check_available_pvs
+from ocs_ci.ocs.cluster import check_ceph_health_after_add_capacity
+
+
+log = logging.getLogger(__name__)
+
+
+@ui
+@tier1
+@skipif_no_lso
+@ignore_leftovers
+@bugzilla("1943280")
```

Comment:
From the bugzilla it looks like we should add more than 3 OSD at a time during scale up but in the test script below I see you are adding only one osd. Am I missing anything, Can you please check?

Reply:
In order to save resources, I added only one disk. I think that I test flexible scaling via UI in this method.
@PrasadDesala @nehaberry What do you think?

Reply:
ack